src/MovieCard/MovieCard.js

```diff
@@ -1,10 +1,40 @@
-import React from 'react';
+import React, { Component } from 'react';
+import './MovieCard.css';
+import CharacterCard from '../CharacterCard/CharacterCard';
+import { createCharacterList } from '../apiCalls';
 
-const MovieCard = () => {
-return (
-  <h1>card</h1>
-)
+class MovieCard extends Component {
+  constructor() {
+    super();
+    this.state = {
+      showCharacterCard : false,
+      characters : []
+    }
+  }
 
-}
+  componentDidMount() {
+    this.setState({ characters : createCharacterList(this.props.characters)})
+  }
+
+  viewCharacter = () => {
+    this.setState({ showCharacterCard : !this.state.showCharacterCard})
+  }
 
+  render() {
+    return (
+      <div className='movie-card'>
+        <h2>{this.props.title}</h2>
+        <p>Episode {this.props.episode}</p>
+        <p>Released in {this.props.releaseYear.slice(0, 4)}</p>
+        <button onClick={this.viewCharacter} className='characters-button'>View Characters</button>
+        {this.state.showCharacterCard ? 
```

Comment:
The logic here is that by making a state key of showCharacterCard set to false, when the button is clicked it will switch to true - and render the character card

Reply:
Great start